src/main/java/com/devqoo/backend/auth/jwt/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,51 @@
+package com.devqoo.backend.auth.jwt;
+
+import static com.devqoo.backend.auth.jwt.SecretKeyType.ACCESS;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+
+    private final JwtProvider jwtTProvider;
+
+    @Override
+    protected void doFilterInternal(
+        HttpServletRequest request, HttpServletResponse response, FilterChain filterChain
+    ) throws ServletException, IOException {
+
+        String token = extractJwtFromHeader(request);
+        
+        if (StringUtils.hasText(token) && jwtTProvider.validateToken(token, ACCESS)) {
+
+            Authentication authentication = jwtTProvider.getAuthentication(token, ACCESS);
+            SecurityContextHolder.getContext().setAuthentication(authentication);
+        }
+
+        filterChain.doFilter(request, response);
+    }
+
+    // Authorization 헤더에서 Token 추출
+    private String extractJwtFromHeader(HttpServletRequest request) {
+
+        String header = request.getHeader("Authorization");
+
+        String bearer = "Bearer ";
+        if (header != null && header.startsWith(bearer)) {
```

Comment:
```suggestion
        if (StringUtlis.hasTest(header) && header.startsWith(bearer)) {
```

위에 30번 줄처럼 스프링이 제공하는 `StringUtils` 변경 요청합니다!

Reply:
그러네요!! 위에서는 저런 형태로 했는데 수정하겠습니다! 👍